Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM alpine:latest
 
 # Version of Radicale (e.g. 2.0.0)
-ENV VERSION master
+ARG VERSION=master
+ENV VERSION ${VERSION}
```

Comment:
Keeping the ```ENV``` part allows to have the ```VERSION``` environment variable available during runtime. So yeah, you're right, I think it's superfluous too.